src/frontend/sharedComponents/BottomSheetModal/Content.tsx

```diff
@@ -108,9 +121,6 @@ const styles = StyleSheet.create({
     fontSize: 20,
     textAlign: "center",
   },
-  buttonsContainer: {
-    paddingHorizontal: 20,
-  },
```

Comment:
this change will break other consumers of the bottom sheet modal component. based on what's described in the [design spec](https://www.notion.so/digidem/d2b2903e77fc4a8cae6d6a8467e25b59?v=def381515abe45578065598c05682c75&p=f28600ee2800443a8bf0980e011933ba&pm=c), this shouldn't be changed. if there's a spacing issue when using this, probably should update the consumer's styles instead of this one

Reply:
Ok, ill override it on my component

Reply:
Never mind, ill just leave the design spec. It is causing the text to be over 2 lines (but i have a zoomed in font on my phone). But i think that is better than overriding the design spec

Reply:
ah interesting, wonder how to address that given translations and all, aside from using more terse language. think those are questions for the design spec in general though :)